how-to/add-mods/curseforge-app.md

```diff
@@ -0,0 +1,31 @@
+---
+hidden: true
+---
+
+# CurseForge App
+
+{% tabs %}
+{% tab title="Through the CurseForge App" %}
+1. Open the **CurseForge App**
+2. Select **Minecraft** from the grid or from the sidebar
+3. In **My Modpacks**, click on **Fabulously Optimized**
+4. Click on the **three dots** button
```

Comment:
```suggestion
4. Click on the **ⵗ** (three dots) button
```
May need a bigger font to be more visible, not sure
Can also try `⋮` or `⫶` (yes, they are different haha)